src/normal.c

```diff
@@ -413,55 +415,68 @@ void sub_81159B4(struct Sprite *sprite)
     sprite->callback(sprite);
 }
 
-void sub_8115A04(u8 taskId)
+// Task data for AnimTask_BlendColorCycle
+#define tPalSelector   data[0] // arg0
```

Comment:
These `// arg*` comments don't seem helpful to me.  It's obvious when reading the function.